playwright/.github/workflows/playwright.yml

```diff
@@ -0,0 +1,26 @@
+name: Playwright Tests
```

Comment:
Why this file is a duplicated of the previous? Remove one of them, only one should be required.